doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,52 +394,116 @@ ZTA principles applied to Cloud infrastructure components are the following:
 
-Open Source Software Security
------------------------------
+Software Supply Chain Security
+------------------------------
+
+Software supply chain attacks are increasing worldwilde and can cause serious damages. Many entreprises and 
+organisations are experiencing these threats. Software suppliers can be targeted and by rebound effect their 
+customers, as seen with the backdoors affecting SolarWinds products and detailed in
+`Defending against SolarWinds attacks 
+<https://www.techtarget.com/searchsecurity/news/252494495/Defending-against-SolarWinds-attacks-What-can-be-done>`_.
+Open-source code weaknesses can also be utilised by attackers, the Log4J vulnerability, impacting lots of 
+applications, is a recent example in this field. The cyber security of software supply chain are often not 
+taken into account. Governments are alerting and requesting actions to face these risks. The British 
+government is hardening the law and standards of cyber security for the supply chain. The US government 
+requested actions to enhance the software supply chain security. The security of software supply chain 
+is a challenge pointed out by the European Network and Information Security Agency, ENISA, 
+in the report `NFV Security in 5G - Challenges and Best Practices 
+<https://www.enisa.europa.eu/publications/nfv-security-in-5g-challenges-and-best-practices>`_.
+ 
+
+Software security
+~~~~~~~~~~~~~~~~~
 
-Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments. Open source code is present in Cloud Infrastructure software from host Operating System to virtualisation layer components, the most obvious being represented by Linux, KVM, QEMU, OpenStack, and Kubernetes. Workloads components can also be composed of open source code. The proportion of open source code to an application source code can vary. It can be partial or total, visible or not. Open source code can be upstream code coming directly from open source public repositories or code within a commercial application or network function. To ensure the security of the whole system, all software and hardware components must reach the same level of security by following best security practices including secure lifecycle management. The SAFECode paper “Managing Security Risks Inherent in the Use of Third-party Components” provides a detailed risk management approach.
+Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments
+with a greater attack surface. All software components must be trusted, from commercial software, open-source 
```

Comment:
The sentence, "Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments with a greater attack surface", can be read as that this is only an issue where the environments have a "greater attack surface". 

One suggestion:
"Software supply chain safety is crucial and is made complex by the greater attack surface provided by the many different supply chains in virtualised, containerised, and edge environments."

Reply:
Thanks, It is better, added